docs/layers.md

```diff
@@ -56,19 +56,19 @@ For features in the `boundaries` layer, there are two additional variants `name:
 
 We populate two fields for Chinese variants:
 
-* `name:zh`: Simplified Chinese
-* `name:zht`: Traditional Chinese
+* `name:zh-hans`: Simplified Chinese
+* `name:zh-hant`: Traditional Chinese
 
-Both `name:zh` and `name:zht` properties are best-effort. It means `name:zh` can sometimes contain Traditional Chinese and `name:zht` can sometimes contain Simplified Chinese. And we don't auto-translate Traditional Chinese to Simplified Chinese or vice versa.
+Both `name:zh-hans` and `name:zh-hant` properties are best-effort. It means `name:zh-hans` can sometimes contain Traditional Chinese and `name:zh-hant` can sometimes contain Simplified Chinese. And we don't auto-translate Traditional Chinese to Simplified Chinese or vice versa.
```

Comment:
Other instances like here of `name:zh-hans` and `name:zh-hant` need that uppercase H, too.

Reply:
> Other instances like here of `name:zh-hans` and `name:zh-hant` need that uppercase H, too.

done